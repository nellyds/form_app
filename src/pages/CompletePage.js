import React from 'react';
import OrderComplete from "../components/orderComplete";
function CompletePage(){
    return(
        <div>
            <OrderComplete />
        </div>
    )
}
export default CompletePage;