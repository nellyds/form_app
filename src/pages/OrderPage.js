import React from 'react';
import OrderInfoForm from "../components/orderInfoForm"


function OrderPage(){
    return(
        <div>
           <OrderInfoForm />
        </div>
    )
}
export default OrderPage;