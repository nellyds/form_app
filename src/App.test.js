import React from 'react';
import { render } from '@testing-library/react';
import CompletePage from "./pages/CompletePage";
import OrderPage from "./pages/OrderPage";
import { shallow } from 'enzyme';
test('renders without crashing', () => {
  expect(
    JSON.stringify(
      Object.assign({}, OrderPage, { _reactInternalInstance: 'censored' }),
    ),
  ).toMatchSnapshot();
});

test('renders without crashing', () => {
  expect(
    JSON.stringify(
      Object.assign({}, CompletePage, { _reactInternalInstance: 'censored' }),
    ),
  ).toMatchSnapshot();
});
