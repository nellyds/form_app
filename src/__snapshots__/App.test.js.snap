// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Confirmation Page without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;

exports[`renders Confirmation Page without crashing 2`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;

exports[`renders Order Page without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;

exports[`renders Order page without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;

exports[`renders app without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;

exports[`renders order page without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;

exports[`renders without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;

exports[`renders without crashing 2`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;
